numba/tests/test_fancy_indexing.py

```diff
@@ -176,6 +176,52 @@ def test_setitem_array(self):
         indices = self.generate_advanced_indices(N)
         self.check_setitem_indices(arr, indices)
 
+    def test_setitem_0d(self):
+        # Test setitem with a 0d-array
+        pyfunc = setitem_usecase
+        cfunc = jit(nopython=True)(pyfunc)
+
+        x1 = np.zeros(3)
+        v1 = np.array(3.14)
+        exp1 = pyfunc(x1.copy(), 0, v1)
+        got1 = cfunc(x1.copy(), 0, v1)
+        self.assertPreciseEqual(exp1, got1)
```

Comment:
`setitem_usecase` returns `None`, is this just asserting `None == None` ?
https://github.com/numba/numba/blob/07bad806dfaee4f764ed1d7407885da1105cf790/numba/tests/test_fancy_indexing.py#L15-L16

Reply:
You're right! I've updated the test to compare the arrays